releasenotes/notes/inst-map-functions-84b176b4a14df449.yaml

```diff
@@ -0,0 +1,17 @@
+---
+ features:
+   - |
+     It is now possible to insert a callable into a :class:`qiskit.pulse.InstructionScheduleMap`
+     which returns a new :class:`qiskit.pulse.Schedule` when it is called with parameters.
+     For example:
+
+     .. code::
```

Comment:
shouldn't this be: `code-block`? I'm not familiar with the `code` directive in sphinx.

Reply:
you're right.